test/generatorAndSchedule.js

```diff
@@ -0,0 +1,403 @@
+
+'use strict';
+
+const assert = require('chai').assert;
+let generator = require('../src/backend/generator.js');
+let app = require('../src/app');
+let webdriver = require('selenium-webdriver');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['webdriver' is never reassigned. Use 'const' instead. (prefer-const)](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)